src/ptc/util/misc.clj

```diff
@@ -144,6 +144,16 @@
       (throw (IllegalArgumentException. (format "Bad GCS URL: '%s'" url))))
     [bucket (or object "")]))
 
+(defn gs-object-leaf
+  "Return object LEAF from a gs://bucket/object URL."
+  [url]
+  (-> (parse-gs-url url) second (str/split #"/") last))
```

Comment:
Isn't that just `(last (str/split #"/" url))`?

Nothing GS or cloudy about it.